lazyflow/operators/ioOperators/opStreamingH5N5Reader.py

```diff
@@ -50,30 +54,59 @@ class OpStreamingHdf5Reader(Operator):
     OutputImage = OutputSlot()
 
     H5EXTS = ['.h5', '.hdf5', '.ilp']
+    N5EXTS = ['.n5']
+
+    # monkeypatch z5py to have it similar enough to be used like h5py
+    def z5pyReadDirectWrapper(self, dest, source_sel=None, dest_sel=None):
+        start = [s.start for s in source_sel]
+        stop = [s.stop for s in source_sel]
+        dest[...] = self.read_subarray(start, stop)
+
+    def z5pyWriteDirectWrapper(self, source, source_sel=None, dest_sel=None):
+        start = [s.start for s in dest_sel]
+        self.write_subarray(start, source)
+
+    z5py.dataset.Dataset.compression = None
```

Comment:
The `compression` attribute is missing in the `z5py.Dataset` API. I will fix that.
However, I don't quite get why you set it to `None` here, is that just a hack to have something?

Reply:
if the h5 dataset is comressed, ilastik reads with a multi-process access object. Setting it to None avoids that. I will change this in ilastik so this only applies to h5